conf/machine/raspberrypi3-64.inc

```diff
@@ -0,0 +1,56 @@
+# Raspberry Pi profile
+# Copyright (c) Cybertrust Japan Co., Ltd.
+#
+# SPDX-License-Identifier: MIT
+#
+
+DISTRO_ARCH ?= "arm64"
+
+# Raspberry Pi needs non-free-firmware repository.
+DISTRO_APT_SOURCES += "conf/distro/emlinux-bookworm-raspberrypi.list"
```

Comment:
This list doesn't depend on RPi, so more generic name like emlinux-bookworm-full.list is preferrable.

Reply:
Fixed here.

https://github.com/miraclelinux/meta-emlinux/compare/b42989cea99efb1d6e43592daa3873e1a25b8a7b..7b0858526111746955445c2372b17b4a6d545598

Reply:
Thanks!